lib/index.js

```diff
@@ -1,30 +1,30 @@
 'use strict';
 
-var Promise         = require('bluebird');
-var PopplerDocument = require('poppler-simple').PopplerDocument;
-var utils           = require('./utils');
+const Promise         = require('bluebird'); // eslint-disable-line no-shadow
```

Comment:
Why do we need to add the eslint rule here?

Reply:
+1?

Reply:
it's overriding the global `Promise` . Which is what we want to do, but an eslint error since it's not usually recommended.

Reply:
IMO we shouldn't be using Bluebird at all in which case we can remove that line, but that's a conversation for another time.